### DISCUSSION TITLE
Content Security Policy is inoring internal whitelisting.

### DISCUSSION THREAD

**Discussion by Eclipse-Computing** [Dotnet-Boxed/Templates#129] [Q&A]
I have opened up a Stack Overflow post with this issue: [http://stackoverflow.com/questions/39541463/](http://stackoverflow.com/questions/39541463/) seems that even though core Google products are supposed to be whitelisted (ReCaptcha v2, etc.), I am unable to get the Content Security Policy to whitelist these services.

I find this to be a breaking issue with something that should be fundamentally simple to do. My inability to get ReCaptcha v2 up and running - despite its services being whitelisted by default - means that I am seriously recommending against any use of this boilerplate within the organization I work for.

**Comment by RehanSaeed** [comment]
I've answered the question on StackOverflow.
